lib/components/viewers/stop-schedule-table.js

```diff
@@ -122,16 +122,19 @@ class StopScheduleTable extends Component {
               ? nextStopTime === highlightedStopTime
               : highlightRow
             const routeName = route.shortName ? route.shortName : route.longName
-            const formattedTime = formatSecondsAfterMidnight(stopTime.scheduledDeparture, timeFormat)
-
+            // Convert to unix (millisceonds) for FormattedTime
+            var now = new Date()
+            var startOfDay = new Date(now.getFullYear(), now.getMonth(), now.getDate())
```

Comment:
These should be `const`.

Reply:
The method here has changed, but using a `const` in https://github.com/opentripplanner/otp-react-redux/pull/480/commits/2ccefb263ad8863edab72e9d796596beda7336f0